library/src/main/java/com/novoda/downloadmanager/lib/DownloadProvider.java

```diff
@@ -520,7 +520,7 @@ private void notifyDownloadStatusChanged() {
         getContext().getContentResolver().notifyChange(downloadsUriProvider.getDownloadsWithoutProgressUri(), null);
     }
 
-    private void notifyBatchesStatusChanged() {
+    private void notifyBatchesChanged() {
```

Comment:
Actually. I'm more wondering why we only notify the `withoutProgress` uri and not the `allBatches` uri as well? Is there a reason